modules/launching-jupyter-and-starting-a-notebook-server.adoc

```diff
@@ -32,15 +32,16 @@ If you credentials are accepted, the *Notebook server control panel* opens displ
 +
 This is not required if you have previously opened Jupyter.
 
-.. Select the *Notebook image* to use for your server.
+.. From the *Notebook image* section, select the *Notebook image* to use for your server.
 .. If the notebook image contains multiple versions, select the version of the notebook image from the *Versions* section.
 +
 [NOTE]
 --
 When a new version of a notebook image is released, the previous version remains available and supported on the cluster. This gives you time to migrate your work to the latest version of the notebook image.
 --
-.. Select the *Container size* for your server.
-.. Optional: Select the *Number of GPUs* (graphics processing units) for your server.
+.. From the *Container size* list, select a suitable container size for your server.
+.. Optional: From the *Accelerator* field, select an accelerator from the list. 
```

Comment:
Same as comment above RE: list elements & repetition.

Could cut down to:
'From the *Accelerator* list, select an accelerator.'